src/module.js

```diff
@@ -1,4 +1,5 @@
 const ExtractTextPlugin = require('extract-text-webpack-plugin');
+const resolveFilePathIfExists = require('./utils').resolveFilePathIfExists;
```

Comment:
if
```const { resolveFilePathIfExists } = require('./utils');```
does not work, then I'd prefer this:
```const resolveFilePathIfExists = require('./utils/resolve-file-path-if-exists');```

Reply:
done